lib/aruba/environment/basic_environment.rb

```diff
@@ -0,0 +1,129 @@
+module Aruba
```

Comment:
So WindowsEnvironment -> WindowsEnvironments
 PosixEnvironment -> PosixEnvironments
BasicEnvironment -> BasicEnvironments  
what about Aruba::Environment ? also make plural?   (Doing so will mean touching code elsewhere, of course)